conftest.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pytest
+
+import release
+
+
+@pytest.fixture
+def release_config():
+    if not os.path.exists('dcos-release.config.yaml'):
+        pytest.skip("Skipping because there is no configuration in dcos-release.config.yaml")
+    return release.load_config('dcos-release.config.yaml')
+
+
+@pytest.fixture
+def release_config_testing(release_config):
+    if 'testing' not in release_config:
+        pytest.skip("Skipped because there is no `testing` configuration in dcos-release.config.yaml")
+    return release_config['testing']
+
+
+@pytest.fixture
+def release_config_aws(release_config_testing, monkeypatch):
```

Comment:
`monkeypatch` arg kind of indicates that only this method is exercised/ tested in code. 
Perhaps others need to be exercised and tested too.

Reply:
monkeypatch was a holdover from a previous revision and should be removed now. good eye